### PR TITLE
adds pingu theme

### PR DESCRIPTION
**Pull request by swissloaf**
### Intent of your Pull Request

adds the theme of pingu to the lobby music

#### Changelog

:cl:  
rscadd: adds pingu theme
/:cl:

### CODE REVIEW

File: yogstation/code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -20,6 +20,7 @@
 		"https://www.youtube.com/watch?v=GISnTECX8Eg",						// Chris Remo - Space Asshole
 		"https://www.youtube.com/watch?v=w34fSnJNP-4",						// Leslie Fish - Dawson's Chrstian (Singer unknown)
 		"https://www.youtube.com/watch?v=le1eD6I7k4s",						// Ronald Jenkee - Piano Wire
+		"https://www.youtube.com/watch?v=KFQpSp-3APw", 						// Andreas Pfluger - Pingu Season 3 Main Theme
```
Comment (Amelia0010) [inline yogstation/code/controllers/subsystem/ticker.dm:23]:
```suggestion
		"https://www.youtube.com/watch?v=KFQpSp-3APw", 						// Andreas Pflüger & David Hasselhoff - Pingu Season 3 Main Theme
```

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Comment by Mr-Space** [comment]
How does pingu fit in with ss13?

**Comment by Hopekz** [comment]
> How does pingu fit in with ss13?

Listen to the theme.
Gives ss13 vibes to me

File: yogstation/code/controllers/subsystem/ticker.dm (hunk 1 above)
Comment (Amelia0010) [reply yogstation/code/controllers/subsystem/ticker.dm:23]: Do you ever just downvote someone for a factual correction to a comment?

**Pull request closed without merging**